benchmarks/benchmarks/sparse_linalg_svds.py

```diff
@@ -24,8 +31,17 @@ def setup(self, k, problem, solver):
                                 "svds_benchmark_files.npz")
         matrices = np.load(datafile, allow_pickle=True)
         self.A = matrices[problem][()]
+        _, s, _ = svd(self.A.toarray(), full_matrices=False)
+        self.all_singular_values = s.sort()
 
     def time_svds(self, k, problem, solver):
-        # consider k = int(np.min(self.A.shape) * k)
-        np.random.seed(0)
-        svds(self.A, k=k, solver=solver)
+        rng = np.random.default_rng(0)
```

Comment:
I think you can safely initialize the `rng` in `setup`--i.e., `self.rng`? To avoid inclusion in timing.